### ISSUE TITLE
Interface for `get`

### ISSUE THREAD

**Issue by bpoldrack** [psyinfra/onyo#503]
Working on #485, I wonder about `get`'s `--key` option.
As far as I can see, every value given to that switch generates a `Filter` and `get` finds the assets that are matched by all given filters (IOW: It's always an 'AND'). Thinking about possible ways to enhance that concept later in order to have more complex queries, I think the name `keys` isn't good. Say, we want to later have an expression that allows for an 'OR' like "match if key1 matches criterion1 OR if key2 matches criterion2", a way to do that could be to come up with an expression that would be a single filter and additional filters would still be combined with an AND. This could be something like `--keys key1=criterion1||key2=criterion2 key3=criterion3`, providing two values to the `--keys` switch which translates to `(key1=criterion1 or key2=criterion2) and key3=criterion3`. This is just an example. The point is: One could come up with anything as long as every value given to that switch establishes one filter and all such filters are combined via AND.

Long story short: This would imply, that `--keys` is actually not a good name in the long run, since a single filter does not necessarily involve a single key. What it does is to specify matching criteria. Hence, `--match` may be better to allow such additions in the future w/o the name becoming strange. WDYT?

**Comment by bpoldrack** [comment]
Never mind. That was stupid. I confused `--keys` with `--filter` *facepalm*
